resume-ab-repo-cassandra/src/main/kotlin/ru/otus/otuskotlin/resume/repo/cassandra/ResumeCassandraDTO.kt

```diff
@@ -0,0 +1,89 @@
+package ru.otus.otuskotlin.resume.repo.cassandra
+
+import com.datastax.oss.driver.api.core.type.DataTypes
+import com.datastax.oss.driver.api.mapper.annotations.CqlName
+import com.datastax.oss.driver.api.mapper.annotations.Entity
+import com.datastax.oss.driver.api.mapper.annotations.PartitionKey
+import com.datastax.oss.driver.api.querybuilder.SchemaBuilder
+import ru.otus.otuskotlin.resume.backend.common.models.*
+
+@Entity
+class ResumeCassandraDTO (
+    @CqlName(COLUMN_ID)
+    @PartitionKey
+    var id: String? = null,
+    @CqlName(COLUMN_FIRST_NAME)
```

Comment:
в таком виде аннотация не сработает. Нужно добавить:
```suggestion
    @field:CqlName(COLUMN_FIRST_NAME)
```